src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,17 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user == null) {
+            System.out.println("Error, no such email address");
+            return false;
+        }
+        if (user.getPassword().equals(password)) {
+            return true;
+        } else {
+            System.out.println("Error, wrong password");
+            return false;
+        }
```

Comment:
```suggestion
        } 
        System.out.println("Error, wrong password");
        return false;
```
we don't need else after return